team/bundles/org.eclipse.jsch.ui/src/org/eclipse/jsch/internal/ui/preference/PreferencePage.java

```diff
@@ -89,8 +91,13 @@ public class PreferencePage extends org.eclipse.jface.preference.PreferencePage
 		implements IWorkbenchPreferencePage{
 
 	private static final String SSH2_PREFERENCE_PAGE_CONTEXT="org.eclipse.jsch.ui.ssh2_preference_page_context"; //$NON-NLS-1$
-	private static final int RSA_KEY_SIZE = 4096;
+	private static final int RSA_KEY_SIZE = 15360;
 	private static final int DSA_KEY_SIZE = 3072;
```

Comment:
Let's add a `MAX` segment in these names to reflect that these are the maximum values:
```suggestion
	private static final int RSA_MAX_KEY_SIZE = 15360;
	private static final int DSA_MAX_KEY_SIZE = 3072;
```